bundles/binding/org.openhab.binding.plclogo/src/main/java/Moka7/IntByRef.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * Moka7 librray is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Library or Lesser General Public License
+ * version 3.0 (LGPLv3) as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
```

Comment:
> Should we build an jar and simply use it? We must add a reference to Moka7 homepage in Wiki and Readme. In my opinion, this way should be ok. Disadvantage of the solution is, that we use own-build jar.

Sorry I still don't have an answer.  When @kaikreuzer has a chance to comment, we will know!  In the meantime, could you squash your 148 commits into one, and in future rebase your branch to current master instead of merging master into your branch?